helm/charts/determined/values.yaml

```diff
@@ -178,7 +178,7 @@ checkpointStorage:
   # storage beyond initial testing as most Kubernetes cluster nodes do not have a shared file
   # system.
   type: shared_fs
-  hostPath: /checkpoints
+  hostPath: /tmp/checkpoints
```

Comment:
Note: I'm doing this because modern k8s won't let you just create a hostPath somewhere like `/`. This is required since the change to use containerd as the default: https://kubernetes.io/blog/2022/02/17/dockershim-faq/

This format is also a little more explicit about what's really happening here and why it's not recommended long term.